### PR TITLE
docs(instances): change rescue mode

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  paragraph: This page helps you to switch your Instance to the new rescue mode
+tags: rescue rescuemode bootscript boot-script instance boot script
+dates:
+  validation: 2023-05-04
+  posted: 2023-05-04
+categories:
+  - compute
+---
+
+The old, bootscript-based, rescue mode will be phased-out on May 31, 2023. To ensure the correct functioning of your Instances, please make sure to switch them to local boot or the new rescue mode before this date.
```
Comment (bene2k1) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:16]: The previous Instances rescue mode, which relied on bootscripts, will be gradually discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: Changing the rescue mode of your Instance
+  description: This page helps you to switch your Instance to the new rescue mode
```
Comment (nedaehm) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:4]:
```suggestion
  description: This page helps you switch your Instance to the new rescue mode
```

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+content:
+  h1: Changing the rescue mode of your Instance
+  paragraph: This page helps you to switch your Instance to the new rescue mode
```
Comment (nedaehm) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:7]:
```suggestion
  paragraph: This page helps you switch your Instance to the new rescue mode
```

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+The previous Instances rescue mode, which relied on bootscripts, will be gradually discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.
+
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [an Instance](/compute/instances/how-to/create-an-instance) using the old (bootscript-based) rescue mode
```
Comment (bene2k1) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:23]:
```suggestion
  - You have [an Instance](/compute/instances/how-to/create-an-instance) using the previous (bootscript-based) rescue mode
```

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+</Message>
+
+
+1. Click **Instances** in the **Compute** section of the side menu in the console. The list of your Instances displays.
+2. Click the name of the Instance that is using the previous rescue mode. The Instance overview displays.
+3. Stop the Instance if it is currently running by using the toggle at the right hand top of the Instances overview.
+3. Click the **Advanced Settings** tab.
```
Comment (ofranc) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:30]: numbering wrong :)

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  paragraph: This page helps you to switch your Instance to the new rescue mode
+tags: rescue rescuemode bootscript boot-script instance boot script
+dates:
+  validation: 2023-05-04
+  posted: 2023-05-04
+categories:
+  - compute
+---
+
+The previous Instances rescue mode, which relied on bootscripts, will be gradually discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.
```
Comment (ofranc) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:16]:
```suggestion
The previous Instances rescue mode, which relied on bootscripts, will be discontinued by May 31, 2023. To ensure that your Instances operate correctly, make sure to switch them to either local boot or the new rescue mode before this deadline.
```

**Review by ofranc — APPROVED** [review]

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [an Instance](/compute/instances/how-to/create-an-instance) using the previous (bootscript-based) rescue mode
+</Message>
+
+
+1. Click **Instances** in the **Compute** section of the side menu in the console. The list of your Instances displays.
+2. Click the name of the Instance that is using the previous rescue mode. The Instance overview displays.
+3. Stop the Instance if it is currently running by using the toggle at the right hand top of the Instances overview.
```
Comment (ldecarvalho-doc) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:29]:
```suggestion
3. Stop the Instance if it is currently running by using the top right toggle of the Instances overview page.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: compute/instances/troubleshooting/change-rescue-mode.mdx (hunk 6 above)
Comment (RoRoJ) [reply compute/instances/troubleshooting/change-rescue-mode.mdx]: "Instance rescue mode" singular

File: compute/instances/troubleshooting/change-rescue-mode.mdx [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+  - You have [an Instance](/compute/instances/how-to/create-an-instance) using the previous (bootscript-based) rescue mode
+</Message>
+
+
+1. Click **Instances** in the **Compute** section of the side menu in the console. The list of your Instances displays.
+2. Click the name of the Instance that is using the previous rescue mode. The Instance overview displays.
+3. Stop the Instance if it is currently running by using the top right toggle of the Instances overview page.
+4. Click the **Advanced Settings** tab.
+5. Scroll to the **Boot Mode** section and change the boot mode of your Instance to **local boot**.
+6xxx. Power the Instance on again by using the toggle.
```
Comment (bene2k1) [inline compute/instances/troubleshooting/change-rescue-mode.mdx:32]:
```suggestion
6. Power the Instance on again by using the toggle. 
```

**Pull request merged**